tests/Aspire.Playground.Tests/Aspire.Playground.Tests.csproj

```diff
@@ -17,11 +17,17 @@
     <PlaygroundSourceDir>$(MSBuildThisFileDirectory)..\..\playground\</PlaygroundSourceDir>
     <TestsSharedDir>$(MSBuildThisFileDirectory)..\Shared\</TestsSharedDir>
     <RunSettingsFilePath>$(MSBuildThisFileDirectory).runsettings</RunSettingsFilePath>
+    <!-- Skip emulators that don't start consistently when running in CI. -->
```

Comment:
NIT: I know you did it on the facts, but might be good to also add the link to the issue tracking removing this in here too. Also not block on this, if you agree we can add it in your next set of changes.